src/plone/restapi/services/site/get.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from plone.registry.interfaces import IRegistry
+from plone.restapi.services import Service
+from Products.CMFPlone.interfaces import IImagingSchema
+from Products.CMFPlone.interfaces import ISiteSchema
+from Products.CMFPlone.utils import getSiteLogo
+from zope.component import getMultiAdapter
+from zope.component import getUtility
+
+
+class SiteGet(Service):
```

Comment:
This should also be exposed as an expander.

Reply:
We also need the closest language root information.

Reply:
> We also need the closest language root information.

you mean apart from the @navroot? To solve cases like /en/some-folder/navroot-folder and to be able to get the 'en' ?

Reply:
> This should also be exposed as an expander.

Added as an expandable endpoint.